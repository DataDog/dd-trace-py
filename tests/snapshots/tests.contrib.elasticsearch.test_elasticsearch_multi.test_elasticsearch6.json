[[
  {
    "name": "elasticsearch.query",
    "service": "elasticsearch",
    "resource": "PUT /ddtrace_index",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "elasticsearch",
    "meta": {
      "_dd.p.dm": "-0",
      "elasticsearch.method": "PUT",
      "elasticsearch.params": "ignore=400",
      "elasticsearch.url": "/ddtrace_index",
      "runtime-id": "87b450ba30ec4f2e8fd734e998aaaa85"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 1309
    },
    "duration": 97388500,
    "start": 1632159160791219000
  }],
[
  {
    "name": "elasticsearch.query",
    "service": "elasticsearch",
    "resource": "DELETE /ddtrace_index",
    "trace_id": 1,
    "span_id": 1,
    "parent_id": 0,
    "type": "elasticsearch",
    "meta": {
      "_dd.p.dm": "-0",
      "elasticsearch.method": "DELETE",
      "elasticsearch.params": "ignore=%5B400%2C+404%5D",
      "elasticsearch.url": "/ddtrace_index",
      "runtime-id": "87b450ba30ec4f2e8fd734e998aaaa85"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 1309
    },
    "duration": 106286200,
    "start": 1632159160889071100
  }]]
