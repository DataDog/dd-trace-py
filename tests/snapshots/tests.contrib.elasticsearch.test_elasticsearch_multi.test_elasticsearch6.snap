[[{"name" "elasticsearch.query"
   "service" "elasticsearch"
   "resource" "PUT /ddtrace_index"
   "type" "elasticsearch"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1615404365434553000
   "duration" 32898000
   "meta" {"runtime-id" "0a66135faff745d78feb7c40c4f70869"
           "elasticsearch.method" "PUT"
           "elasticsearch.url" "/ddtrace_index"
           "elasticsearch.params" "ignore=400"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 51995
              "_dd.tracer_kr" 1.0}}]
 [{"name" "elasticsearch.query"
   "service" "elasticsearch"
   "resource" "DELETE /ddtrace_index"
   "type" "elasticsearch"
   "error" 0
   "span_id" 0
   "trace_id" 1
   "parent_id" nil
   "start" 1615404365467709000
   "duration" 27295000
   "meta" {"runtime-id" "0a66135faff745d78feb7c40c4f70869"
           "elasticsearch.method" "DELETE"
           "elasticsearch.url" "/ddtrace_index"
           "elasticsearch.params" "ignore=%5B400%2C+404%5D"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 51995
              "_dd.tracer_kr" 1.0}}]]
